src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="conteiner">
+      <div class="Product">
+        <div class="Product__img" href="#">
+          <img src="images/imac.jpeg" 
+               alt="APPLE A1419 iMac 27 "
+               value="APPLE A1419 iMac 27 Retina 5K (MNED2UA)"
+          >
+        </div>
+        <div class="Product__name">
+          <h1>
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+        </div>
+        <div class="Product__code">
+          <h2>
+            Product code: 195434
+          </h2>
+        </div>
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <h3>
+            Reviews: 5
+          </h3>
+        </div>
```

Comment:
copy this block from `stars` task, it should have modifier stars--{n} and number of active stars should be changed accordingly to